cockatrice/src/chatview.cpp

```diff
@@ -73,10 +73,18 @@ void ChatView::appendHtml(const QString &html)
         verticalScrollBar()->setValue(verticalScrollBar()->maximum());
 }
 
-void ChatView::appendHtmlServerMessage(const QString &html)
+void ChatView::appendHtmlServerMessage(const QString &html, bool optionalIsBold, QString optionalFontColor)
 {
     bool atBottom = verticalScrollBar()->value() >= verticalScrollBar()->maximum();
-    prepareBlock().insertHtml("<font color=" + SERVER_MESSAGE_COLOR + ">" + html + "</font>");
+
+    QString htmlText = html;
```

Comment:
Being picky, i see no reason why this is assigned and then re-assigned on the next line. You could save some cpu cycles by:
```
QString htmlText = "<font color=" + ((optionalFontColor.size() > 0) ? optionalFontColor : SERVER_MESSAGE_COLOR) + ">" + html + "</font>";
```